src/Orc.Controls/Controls/Callout/Win32/Rect.cs

```diff
@@ -0,0 +1,105 @@
+﻿namespace Orc.Controls.Win32
```

Comment:
Since this is win32, we should move it to a shared namespace in case we need win32 in other components. So move file to Orc.Controls/Win32